apps/showcase/src/assets/trainings/sdk/steps/model-extension/exercise/core/flight/flight.ts

```diff
@@ -0,0 +1,9 @@
+/* TODO create the type FlightCoreIfy which extends Flight, imported from the ../base folder */
```

Comment:
We could also have the example of a usage of the id in the UI (something as stupid as an alert of a flight id) to show that it is completely transparent to the application (Flight is still exposed with the same name etc.) ?

Reply:
We are already displaying the revived object 
![image](https://github.com/user-attachments/assets/503247ba-0040-4f98-baa5-39713c857124)
![image](https://github.com/user-attachments/assets/f463bed0-a118-4bc3-949e-7ef37cd5aaa1)

Do you think we need more than that?

Reply:
My bad, I did not look the shared file into details and missed that :)